### PR TITLE
补充thrift-http 映射注解规范

### PR DESCRIPTION
**Pull request by wangjingpei**
补充thrift-http 映射注解规范

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
Please  fix those typos in this PR.

**Review by YangruiEmma — APPROVED** [review]

File: content/en/docs/kitex/Tutorials/advanced-feature/generic-call/thrift_idl_annotation_standards.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+}
+struct BizResponse {
+    1: optional string T  (api.header= 'T') 
+    // This field will be filled in the header returned to the client
+    2: optional map<i64, RspItem> rsp_items  (api.body='rsp_items')
+    // first level key = 'rsp_items'
+    3: optional i32 v_enum  (api.none = 'true') // Ignore current parameter
+    4: optional list<RspItem> rsp_item_list  (api.body = 'rsp_item_list')
+    // The business specifies the HTTP Code itself. If not specified, baseResp.StatuCode=0 -> HTTPCode=200,  other HTTPCode=500      
+    5: optional i32 http_code  (api.http_code = 'http_code')
```
Comment (jayantxie) [inline content/en/docs/kitex/Tutorials/advanced-feature/generic-call/thrift_idl_annotation_standards.md:110]: optional i32 http_code                         (api.http_code = '')

File: content/en/docs/kitex/Tutorials/advanced-feature/generic-call/thrift_idl_annotation_standards.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+struct RspItem{
+    1: optional i64 item_id // By default, it is serialized with the field name as key, which is equivalent to using gotag `json:"item_id"`
+    2: optional string text
+}
+struct BizResponse {
+    1: optional string T  (api.header= 'T') 
+    // This field will be filled in the header returned to the client
+    2: optional map<i64, RspItem> rsp_items  (api.body='rsp_items')
+    // first level key = 'rsp_items'
+    3: optional i32 v_enum  (api.none = 'true') // Ignore current parameter
```
Comment (jayantxie) [inline content/en/docs/kitex/Tutorials/advanced-feature/generic-call/thrift_idl_annotation_standards.md:107]: api.none = ''

**Review by jayantxie — APPROVED** [review]
LGTM

**Pull request merged**